packages/suite-desktop-core/src/app.ts

```diff
@@ -128,18 +130,38 @@ const init = async () => {
 
     await app.whenReady();
 
-    // No UI mode with bridge only
+    // Load bridge module first, it is required in both UI and daemon mode
+    const { onLoad: loadBridgeModule, onQuit: quitBridgeModule } = initBridgeModule({ store });
+    await loadBridgeModule();
```

Comment:
I believe bridge doesn't access the internet so Tor shouldn't be needed. 
It's true that if Bridge has some issue starting, it can delay the launch of the app. There is a timeout of 3 seconds and errors are silent, so it should start after that.

Reply:
Oh ok, I thought there's also 3 attempts. So the worst case is waiting 3 seconds for UI, right?

Reply:
Yes I think so